Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -104,45 +116,18 @@ protected override void Awake()
 #if UNITY_WSA && UNITY_2017_2_OR_NEWER
         private void RenderFloorQuad()
         {
-            if (HolographicSettings.IsDisplayOpaque)
-            {
-                // Defaulting coordinate system to RoomScale in immersive headsets.
-                // This puts the origin 0, 0, 0 on the floor if a floor has been established during RunSetup via MixedRealityPortal
-                XRDevice.SetTrackingSpaceType(TrackingSpaceType.RoomScale);
-            }
-            else
-            {
-                // Defaulting coordinate system to Stationary for HoloLens.
-                // This puts the origin 0, 0, 0 at the first place where the user started the application.
-                XRDevice.SetTrackingSpaceType(TrackingSpaceType.Stationary);
-            }
-
-            if (FloorQuad != null && HolographicSettings.IsDisplayOpaque)
+            if (FloorQuad != null && XRDevice.GetTrackingSpaceType() == TrackingSpaceType.RoomScale)
             {
                 floorQuadInstance = Instantiate(FloorQuad);
 
                 if (!XRDevice.isPresent)
                 {
                     // So the floor quad does not occlude in editor testing, draw it lower.
-                    floorQuadInstance.transform.localPosition = new Vector3(0, -3, 0);
+                    floorQuadInstance.transform.position = new Vector3(0, -3, 0);
```

Comment:
Should this be cached as a const?

Reply:
![image](https://user-images.githubusercontent.com/3580640/33040393-c4ad61da-cdef-11e7-975a-2f54592410a6.png)

Reply:
Ah not as a const but a readonly private field.

Reply:
Why are we choosing `-3` anyway?

Reply:
> Why are we choosing -3 anyway?

Experimentation in the editor found that this got it out of the way, and is only used in the case where a headset isn't attached. Otherwise, it functions as normal.

It might not even be worth showing the floor in the editor at an arbitrary location, since it doesn't mean much without a headset attached and a boundary defined.

Reply:
Should we add these comments to our description of the value?